GPSTest/src/main/res/values/strings.xml

```diff
@@ -275,8 +278,12 @@
     <string name="pref_dark_theme_summary">Give the background a dark color and text a light color
     </string>
 
-    <string name="pref_dms_mode_title">Use DMS format</string>
-    <string name="pref_dms_mode_summary">Show latitude and longitude in degrees, minutes, and seconds instead of decimal degrees</string>
+    <string name="pref_preferred_display_mode_title">Coordinate Display Format</string>
```

Comment:
Instead of "Display mode", could you please change the naming conventions for variables to use "Coordinate format" instead?  The name of this preference title can also just be `Coordinate format`.